### PR TITLE
comments & stuff

### PR DESCRIPTION
**Pull request by SansPapyrus683**
_If any of the below doesn't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which includes the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making them.

### CODE REVIEW

File: solutions/usaco-1084.mdx [hunk 1]
```diff
@@ -17,30 +17,34 @@ Note: The video solution might not be the same as other solutions. Code in C++.
 
 ```java
 import java.util.*;
-import java.io.*;
-public class Main {
+
+public class OddPhotos {
 	public static void main(String[] args) {
 	  	Scanner sc = new Scanner(System.in);
```
Comment (envyaims) [inline solutions/usaco-1084.mdx:23]: why still scanner

Comment (SansPapyrus683) [reply solutions/usaco-1084.mdx:23]: matter of preference
just like scanf or cin

Comment (envyaims) [reply solutions/usaco-1084.mdx:23]: @SansPapyrus683 from code conventions lol:
image

Comment (SansPapyrus683) [reply solutions/usaco-1084.mdx:23]: fair enough

**Review by envyaims — APPROVED** [review]
urdidir=t

**Pull request merged**